docs/markdown/Reference-manual.md

```diff
@@ -443,8 +443,9 @@ arguments:
   that override those set in the subproject's `meson_options.txt`
   (like `default_options` in [`project()`](#project), they only have
   effect when Meson is run for the first time, and command line
-  arguments override any default options in build files)
-- `fallback`: specifies a subproject fallback to use in case the
+  arguments override any default options in build files). *(since 0.56.0)*
+  a dictionary of string: string mappings is also accepted.
```

Comment:
nitpicking: I would remove the space `string:string`, maybe it's just me but I was reading `string: string` as a colon in the sentence.